app/_locales/en/messages.json

```diff
@@ -4229,9 +4229,6 @@
   "recipientAddressPlaceholder": {
     "message": "Enter public address (0x) or ENS name"
```

Comment:
```suggestion
    "message": "Enter public address (0x) or domain name"
```

Reply:
I'd prefer we make this change now, because this text is inaccurate and would confuse users.

Reply:
ok, I made the changes to the english text